flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/sink/committables/CommittableCollectorSerializerTest.java

```diff
@@ -107,6 +109,29 @@ void testCommittableCollectorV2SerDe() throws IOException {
                 "Deserialized CommittableCollector", subtaskId, numberOfSubtasks, copy);
     }
 
+    @Test
+    void testAlignSubtaskCommittableManagerCheckpointWithCheckpointCommittableManagerCheckpointId()
+            throws IOException {
+        // Create CommittableCollector holding a higher checkpointId than
+        // Sink.InitContext#INITIAL_CHECKPOINT_ID
+        final CommittableCollector<Integer> committableCollector =
```

Comment:
nit:
Probably will never happen, but what if Sink.InitContext#INITIAL_CHECKPOINT_ID value would be changed in the future?

Maybe defining `checkpointId as Sink.InitContext#INITIAL_CHECKPOINT_ID + 1` or something like that?

Reply:
Good point, I'll use `Sink.InitContext#INITIAL_CHECKPOINT_ID + 1`